cves/kernel/CVE-2020-27673.yml

```diff
@@ -109,66 +109,66 @@ vcc_instructions: |
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: ab9a1cca3d172876ae9d5edb63abce7986045597
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: d0b075ffeede257342c3afdbeadd2fda8504ecee
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: 9a489f45a155fe96b9b55fbbef2b757ef7737cfc
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: 54c9de89895e0a36047fcc4ae754ea5b8655fb9d
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 073d0552ead5bfc7a3a9c01de590e924f11b5dd2
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 7beb290caa2adb0a399e735a1e175db9aae0523a
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: e46cdb66c8fc1c8d61cfae0f219ff47ac4b9d531
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 6cb6537d34a146c8f3e232da9ffd79ae1265fed3
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 38e20b07efd541a959de367dc90a17f92ce2e8a6
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 1fe565517b57676884349dccfd6ce853ec338636
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: f87e4cac4f4e940b328d3deb5b53e642e3881f43
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 7e186bdd0098b34c69fb8067c67340ae610ea499
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 0891fb39ba67bd7ae023ea0d367297ffff010781
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: 348be43384e6bcd5e9da7ff5f1680d49f65c488d
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 229664bee6126e01f8662976a5fe2e79813b77c8
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: f731e3ef02b4744f4d7ca2f63539b900e47db31f
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 0ec53ecf38bcbf95b4b057328a8fbba4d22ef28b
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: b5328cd14557880e9eb757a8a9c8a88f1b23533a
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: c149e4cd08ba01f4d2d0104f469d5f5419294e06
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: ca62ce8cde36cde5a31af8f9bb23e6430700e437
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 94032c506720e26402db64987e08168565b57990
+    :note: "Discovered automatically by archeogit. Manually confirmed."
+  - commit: 9158c3588a0693ef11208f5961f0d6b0241694d8
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: 780f36d8b3fa9572f731d4fb85067b2e45e6f993
+    :note: "Discovered automatically by archeogit. Refactoring."
+  - commit: de88d622fd68bd4dbee0f80168218b23f798fd0
```

Comment:
```suggestion
  - commit: 3de88d622fd68bd4dbee0f80168218b23f798fd0
```

Reply:
Found it!

Reply:
Wow, i did not catch that, thank you! The pipeline still seems to fail though! Is there anyway I can run the checkers locally and get descriptive errors? I don't see anything on readme like the writeup suggests